doc/source/r_interface.rst

```diff
@@ -60,7 +60,7 @@ can be replaced with::
 
     from rpy2.robjects import r
     r.data('iris')
```

Comment:
I would remove everything here except for the top warning, then put a link to the previous page of the built docs.